lib/resources/r4/immunization.yaml

```diff
@@ -27,6 +52,18 @@ fields:
       "status": "completed"
     }
 
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.status
+  description: Indicates the current status of the immunization event.
+  action: update
+  example: |
+    {
+      "status": "not-done"
+    }
+
```

Comment:
for all these duped fields, you should instead be able to do this:
```
  action:
    - create
    - update
```

Reply:
Updated all the fields. https://github.com/cerner/fhir.cerner.com/pull/295/commits/fe9f32d2c48645c5ba462f17793818a3e32b5d4a#diff-2c50e8f41daaba3a640861e7b6e81488L11